RELEASING.md

```diff
@@ -77,28 +79,28 @@ After all [pull requests](https://github.com/GoogleCloudPlatform/google-cloud-ru
-
-1. Confirm that the gem for the new version is available on [RubyGems.org](https://rubygems.org/gems/google-cloud).
+1. Repeat steps 1 through 20 if you are releasing multiple gems.
 
-1. Confirm that the new version is displayed after "Latest release" on the [google-cloud-ruby gh-pages site](http://googlecloudplatform.github.io/google-cloud-ruby/).
+1. Wait until the last tag build job has successfully completed on Circle CI. Then push your commits to the master branch. This will trigger another [Circle CI](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-ruby) build on master branch.
 
-1. Confirm that the new version is displayed in the packages pulldown and the version switcher on the [google-cloud-ruby docs site](https://googlecloudplatform.github.io/google-cloud-ruby/#/). Verify that the new docs version contains the public API changes in the release.
+    ```sh
+    $ git push <remote> master
+    ```
 
-1. Some time later, after they have completed, confirm that [Travis CI (Mac OS X)](https://travis-ci.org/GoogleCloudPlatform/google-cloud-ruby) and [Appveyor CI (Windows)](https://ci.appveyor.com/project/GoogleCloudPlatform/google-cloud-ruby) builds are also green.
+1. After the Circle CI master branch build has successfully completed, confirm that [Travis CI (Mac OS X)](https://travis-ci.org/GoogleCloudPlatform/google-cloud-ruby) and [Appveyor CI (Windows)](https://ci.appveyor.com/project/GoogleCloudPlatform/google-cloud-ruby) master branch builds are also green.
 
-1. If your version change is greater than the [semver](http://semver.org/) patch version, then when you are done releasing all individual packages, you should follow these same instructions to release the `google-cloud` umbrella package. Furthermore, if your major releases included at least one dependency of the `stackdriver` umbrella package (currently those dependencies are `google-cloud-logging`, `google-cloud-trace`, `google-cloud-error_reporting`, and `google-cloud-monitoring`), then you should also follow these same instructions to release the `stackdriver` umbrella package. It is important that the dependencies of the `google-cloud` and `stackdriver` gems remain compatible so the two can co-exist in the same bundle.
+1. If your version change is greater than the [semver](http://semver.org/) patch version, then when you are done releasing all individual packages, you should follow these same instructions to release the `google-cloud` umbrella package. Furthermore, if your major releases included at least one dependency of the `stackdriver` umbrella package (currently those dependencies are `google-cloud-debugger`, `google-cloud-error_reporting`, `google-cloud-logging`, and `google-cloud-trace`), then you should also follow these same instructions to release the `stackdriver` umbrella package. It is important that the dependencies of the `google-cloud` and `stackdriver` gems remain compatible so the two can co-exist in the same bundle.
```

Comment:
I would change this condition to "If you updated `google-cloud/google-cloud.gemspec` or `stackdriver/stackdriver.gemspec` in the previous steps" (or something similar) for clarity. It's unclear what "version change" means here because you may have just released multiple gems.

Reply:
Also, why not just make the `google-cloud`/`stackdriver` release part of the individual gem releases, instead of a separate final step?

Reply:
> why not just make the google-cloud/stackdriver release part of the individual gem releases, instead of a separate final step?

Because of the order, I think they need to be released after their dependencies?

Reply:
Right. If you do something like update a dependency that is not compatible with the previous release, then you want to wait until _all_ the individual gems are released before releasing a new umbrella gem like google-cloud or stackdriver. We have to be very careful to avoid bad dependencies.

Also, it means less releases, which is important since they are so time consuming.

Reply:
Ah, right, I didn't mean "release google-cloud in each iteration of the above loop", I meant to consider treating the google-cloud release as "just another gem being released" to simplify the instructions, instead of calling it out in its own special step. But I buy that it's worth having its own step to ensure that it doesn't accidentally get released too early.

Reply:
@geigerj I think this step needs to be moved up to line 96 (just below the loop). I'll simplify the instructions in this step (using your text), but I won't remove it.